source/Octopus.Tentacle.Contracts/Octopus.Tentacle.Contracts.csproj

```
@@ -28,7 +28,7 @@
     </Otherwise>
   </Choose>
   <ItemGroup>
-    <PackageReference Include="Halibut" Version="8.1.1514" />
+    <PackageReference Include="Halibut" Version="8.1.1592" />
```

Try this version (it looks like it has actually released):

```suggestion
    <PackageReference Include="Halibut" Version="8.1.1591" />
```